docs/accessibility/index.md

```diff
@@ -0,0 +1,60 @@
+---
+id: index
+title: Backstage Accessibility
+description: Documentation on Backstage Accessibility
+---
+
+In an effort to bake accessibility practices further into our process of building features for Backstage, we’ve rolled out automated CI tests to the OSS project on some of our core features, Software Catalog, Software Templates, Search and TechDocs. As these are just a few plugins of many out there, we want to encourage you to consider the accessibility implications of your work on Backstage in order to build a great experience for everyone.
+
+## How to contribute
+
+There are multiple ways to contribute to making Backstage accessible to everyone, below we list examples of some ways you can do that.
+
+### Run Lighthouse in CI on your plugin
+
+If your plugin lives in the [Backstage main repository](https://github.com/backstage/backstage/) you can modify the [urls in the Lighthouse config](https://github.com/backstage/backstage/blob/master/lighthouserc.js#L19-L34) to run the Lighthouse checks on urls where your plugin exists as well. E.g.
+
+```diff
+ ci: {
+    collect: {
+      url: [
++        /** Your plugin paths */
++        'http://localhost:3000/your-plugin-path,
+        /** Software Catalog */
+        'http://localhost:3000/catalog',
+        'http://localhost:3000/catalog-import',
+        'http://localhost:3000/catalog/default/component/backstage',
+        ...
+      ],
+      settings: {
+       ...
+      },
+     ...
+    },
+    assert: {
+      ...
+    },
+  },
+```
+
+To make sure the [Accessibility Github workflow](https://github.com/backstage/backstage/blob/10759b6ad2561bd86183ad940256f9a309c7a6b0/.github/workflows/verify_accessibility.yml) is running when changes are made to your plugin folders, also modify the [list of paths](https://github.com/backstage/backstage/blob/10759b6ad2561bd86183ad940256f9a309c7a6b0/.github/workflows/verify_accessibility.yml#L7-L16).
```

Comment:
IDEA: Maybe change this link to master. https://github.com/backstage/backstage/blob/10759b6ad2561bd86183ad940256f9a309c7a6b0/.github/workflows/verify_accessibility.yml -> https://github.com/backstage/backstage/blob/master/.github/workflows/verify_accessibility.yml .
Also applicable to the other link specifying lines 7 to 16.

Reply:
Yeah its great points, I decided to used permalinks in case the line numbers changed (e.g. new path gets added) to avoid pointing to the wrong part of the config. But let me update the one that don't use line number 👍

Reply:
https://github.com/backstage/backstage/pull/17698/commits/ee740ac2eac79cc6f4ebabff0d9665b2562beffb